openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -171,7 +171,7 @@ def attach_image(img_dict, filename):
     if img_path:
         with open(img_path, 'rb') as img:
             msg_image = MIMEImage(img.read(), name=os.path.basename(img_path))
-            msg_image.add_header('Content-ID', b'<{}>'.format(img_dict['cid']))  # xss-lint: disable=python-wrap-html
+            msg_image.add_header('Content-ID', '<{}>'.format(img_dict['cid']))  # xss-lint: disable=python-wrap-html
```

Comment:
I have to remove `b` otherwise sitepackages throws error.
```
    msg_image.add_header('Content-ID', six.b('<{}>'.format(img_dict['cid'])))  # xss-lint: disable=python-wrap-html
  File "/usr/lib/python3.5/email/message.py", line 544, in add_header
    self[_name] = SEMISPACE.join(parts)
TypeError: sequence item 0: expected str instance, bytes found
```